Site/css/style.css

```diff
@@ -23,7 +22,8 @@ media-queries vil brukes for at større sider blir som beskrevet i oppgaven.*/
 /*over-overskrift, kun brukt på index-siden*/
 h1 {
 	font-weight: bold;
-	color: #f09ca2;
+	color: #fce6de;
+	/* color: #f09ca2; */
```

Comment:
gamle kommentarer burde kanskje fjernes hvis de skal se over CSS filen våre?

Reply:
nå kommenterte jeg mest ut for å sjekke litt frem og tilbake, siden dette er en endring vi beskriver i p3 changes (eller, vi må gjøre det iaf), så tenker jeg at det ikke er farlig om det står :)